extensions/jdbc/jdbc-db2/deployment/src/main/java/io/quarkus/jdbc/db2/deployment/JDBCDB2Processor.java

```diff
@@ -77,4 +78,12 @@ void registerServiceBinding(Capabilities capabilities,
         }
         dbKind.produce(new DefaultDataSourceDbKindBuildItem(DatabaseKind.DB2));
     }
+
+    @BuildStep
+    void addBouncyCastleExportsToNativeImage(BuildProducer<JPMSExportBuildItem> jpmsExports) {
```

Comment:
This was merged a bit too fast, I don't think the method name is what you wanted.

Reply:
Oops, indeed... copy paste... I'll open a new one

Reply:
Missed it, my bad...

Reply:
https://github.com/quarkusio/quarkus/pull/21895